models/models.js

```diff
@@ -0,0 +1,8 @@
+const db = require("../db/connection")
+
+exports.selectAllTopics = () => {
+    const queryValues = []
```

Comment:
We also don't need this line yet as we aren't dealing with any queries as of yet :)